super/pom.xml

```diff
@@ -58,7 +58,8 @@
     <projectreactor.netty.version>1.0.11</projectreactor.netty.version>
     <assertj.version>3.16.1</assertj.version>
     <xmlunit.version>1.6</xmlunit.version>
-    <mockito.core.version>3.6.0</mockito.core.version>
+    <mockito.core.version>4.1.0</mockito.core.version>
+    <mockito.inline.version>4.1.0</mockito.inline.version>
```

Comment:
I wouldn't introduce a separate version for this, looks too tightly coupled to core, so please use that version-variable

Reply:
My initial thoughts as well, added the separate version at the last moment, but reverted this now.